selfdrive/car/subaru/carcontroller.py

```diff
@@ -46,6 +72,49 @@ def update(self, CC, CS):
 
       self.apply_steer_last = apply_steer
 
+    ### LONG ###
+
+    cruise_rpm = 0
+    cruise_throttle = 0
+
+    brake_cmd = False
+    brake_value = 0
+
+    if self.CP.openpilotLongitudinalControl:
+
+      gas, brake = compute_gb(actuators.accel)
+
+      # Manual trigger using wipers signal
+      #if CS.wipers:
+      #  brake = 0.5
+      #  print("wipers set brake 0.5")
+      #  brake_cmd = True
+
+      if CC.longActive and brake > 0:
+        brake_value = clip(int(brake * CarControllerParams.BRAKE_SCALE), CarControllerParams.BRAKE_MIN, CarControllerParams.BRAKE_MAX)
+        brake_cmd = True
+        #print('brake: %s, es_brake_pressure: %s es_brake_active: %s brake_value: %s' % (brake, CS.es_brake_pressure, CS.es_brake_active, brake_value))
+
+      # PCB passthrough
```

Comment:
do we have the dash signal for no AEB? that's what we do for all the other experimental longitudinal cars right now

Reply:
ES_LKAS_State LKAS_Alert_Msg = 6 is Pre_Collision_Braking and ES_LKAS_State LKAS_Alert = 5 is Pre_Collision_Activated_Alert. I'll test if they match with currently used ES_Brake Cruise_Brake_Active

Reply:
I checked that ES_LKAS_State LKAS_Alert = 2 (FCW_Repeated_Beep) also triggers braking (ES_Brake Brake_Pressure) so it looks like ES_Brake Cruise_Brake_Active would be good single indicator for all AEB cases to pass through, since ACC is idle when openpilot long is used.